util/grpcutil: TestShouldPrint failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/1ae28f69319db60c733ea568f9c50260a277cf85

Parameters:
```
TAGS=deadlock
GOFLAGS=
```

Stress build found a failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=236867&tab=buildLog

```
	log_test.go:71: expected second call print=false, got print=true: {foobar=%s [0xc420146280] false}


ERROR: exit status 1

27855 runs completed, 1 failures, over 2m14s
make: *** [stress] Error 1
Makefile:201: recipe for target 'stress' failed
```

--- a/pkg/util/grpcutil/log_test.go
+++ b/pkg/util/grpcutil/log_test.go
@@ -23,7 +23,7 @@ import (
 )
 
 func TestShouldPrint(t *testing.T) {
-	const duration = 10 * time.Millisecond
+	const duration = 100 * time.Millisecond
 
 	formatRe, err := regexp.Compile("^foo")
 	if err != nil {

